src/main/java/com/codegym/jrugotom5/service/AdvertService.java

```diff
@@ -43,17 +43,15 @@ public List<AdvertFullInfoDTO> getAdvertsByDateRange(LocalDate from, LocalDate t
 
     public List<AdvertBasicInfoDTO> getAllAdverts() {
         return Streamable.of(advertRepository.findAll())
-                .map(advert->modelMapper.map(advert, AdvertBasicInfoDTO.class))
+                .map(advert -> modelMapper.map(advert, AdvertBasicInfoDTO.class))
                 .toList();
     }
 
     public List<AdvertInfoForCreatorDto> getAdvertsByUserId(Long id) {
         if (id < 1) {
```

Comment:
What will be result if you will pass id = 1?

Reply:
> What will be result if you will pass id = 1?

it will return all adverts from user with id = 1

Reply:
Yeah, you are right. I overlooked